src/api/index.ts

```diff
@@ -167,25 +173,46 @@ router.get('/:installationId',
     } catch (err) {
       req.log.error(err);
       res.status(500).json(err);
+    } finally {
+      const elapsed = Date.now() - requestStart;
+      const tags = {
+        path:'/:installationId',
+        method: 'GET',
+        status: res.status.toString(),
```

Comment:
In case of timeout we could get it like 500 in metrics. I'd rather make a separate `client_error` status for exceptions

Reply:
Can you clarify this a little more? Do you mean you want to change `status: res.status.toString()` to something like `status: client_error`?